.github/workflows/lts-branch-create-pull-request.yaml

```diff
@@ -40,7 +40,6 @@ jobs:
         run: |
           git config user.name $GIT_USER_NAME
           git config user.email $GIT_USER_EMAIL
-          git config --global --add safe.directory /github/workspace
```

Comment:
What's the affect of removing this line?

Reply:
this was added yesterday as a workaround. the updated `repo-sync/pull-request` version below now has this fix so we can remove the workaround